app/Group.php

```diff
@@ -464,11 +464,6 @@ public function changesShouldPushToZapier()
         return false;
     }
 
-    public function getMaxUpdatedAtDevicesUpdatedAtAttribute()
```

Comment:
Is this definitely not used?  I remember it being a fairly recent addition.

Reply:
Nope - search for MaxUpdatedAtDevicesUpdatedAt finds nowt.